src/mono/mono.proj

```diff
@@ -457,7 +471,7 @@
         <Destination>$(RuntimeBinDir)$(MonoStaticFileName)</Destination>
       </_MonoRuntimeArtifacts>
       <_MonoRuntimeArtifacts Include="$(_MonoAotCrossFilePath)">
-        <Destination>$(RuntimeBinDir)cross\mono-aot-cross</Destination>
+        <Destination>$(RuntimeBinDir)cross\$(TargetOS)-$(TargetArchitecture)\mono-aot-cross</Destination>
       </_MonoRuntimeArtifacts>
```

Comment:
Why is this needed ? Isn't the runtime pack already os/arch specific ?

Reply:
We need a clean way to have multiple cross compilers next to each other, in order to enable a user to build an apk which supports multiple target architectures (which is common on Android).